### PR TITLE
fix: broken link

### LINKED ISSUE #9113: link failure
### MDN URL

https://developer.mozilla.org/fr/docs/Web/CSS/inheritance

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

Hello,
On the 1st line of the page, it is written "Pour chaque propriété CSS, la spécification ...... " with a link for "chaque propriété CSS".
The link does not work (Page not found).

I think the link should point to:
https://developer.mozilla.org/fr/docs/Web/CSS/Reference#keyword_index

### What did you expect to see?

a link that works

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `fr/web/css/inheritance`
* MDN URL: https://developer.mozilla.org/fr/docs/Web/CSS/inheritance
* GitHub URL: https://github.com/mdn/translated-content/blob/main/files/fr/web/css/inheritance/index.md
* Last commit: https://github.com/mdn/translated-content/commit/79faa1509d68be77f14971a843e43da59dedff2f
* Document last modified: 2022-09-21T09:43:31.000Z

</details>

### PR DESCRIPTION
**Pull request by elo1lson**
* fix issue  related in issue #9113

### Description
Broken link
### Related issues and pull requests

Fix #9113

### CODE REVIEW

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
The file is moved here (arquivos/fr/rede) for unknown reasons. Its path should stay the same.

**Comment by SphinxKnight** [comment]
Thanks for stepping in @elo1lson, however the link could be improved and most importantly, the file shouldn't be moved.

**Comment by elo1lson** [comment]
@SphinxKnight sorry, but I believe the `/fr/Reference_CSS` route doesn't exist in any language, I believe the correct one would be `/fr/Css/Reference`

**Comment by SphinxKnight** [comment]
> @SphinxKnight sorry, but I believe the `/fr/Reference_CSS` route doesn't exist in any language, I believe the correct one would be `/fr/Css/Reference`, oh, and sorry, my translator translated the names of the folders by mistake, I noticed that now, I'll fix it

please have a look at my suggestion for the right route

File: arquivos/fr/rede/css/herança/index.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ original_slug: Web/CSS/Héritage
 
 {{CSSRef}}
 
-Pour [chaque propriété CSS](fr/R%c3%a9f%c3%a9rence_CSS), la spécification indique si, par défaut, cette propriété est héritée ou non. Cela permet de définir le comportement qu'on observera lorsqu'aucune valeur n'est spécifiée pour une propriété pour un élément donné.
+Pour [chaque propriété CSS](fr/Css/Reference), la spécification indique si, par défaut, cette propriété est héritée ou non. Cela permet de définir le comportement qu'on observera lorsqu'aucune valeur n'est spécifiée pour une propriété pour un élément donné.
```
[the remark this answers is not in the document]
Comment (SphinxKnight) [reply arquivos/fr/rede/css/herança/index.md]: this has not been taken into account

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
see previous suggestion in the review for the right link to use

**Review by SphinxKnight — CHANGES_REQUESTED** [review]
the file moved again to another location.
@elo1lson I'm going to switch this one to a draft.

**Pull request closed without merging**